### ISSUE TITLE
New command: skrik

### ISSUE THREAD

**Issue by joacimmelin** [punktniklas/NiKom#43]
A command that enables a user to send a online message to all other logged users. If you don't want to be disturbed by these messages you should be able to toggle this via a flag. 

Example: 

(Se) Tiden > Skrik "Vansinnigt tråkigt väder ute!"

Meddelandet skickat till X inloggade medlemmar (X = number of active logins) 

The other users would get the output like this (it has to be shown automatically and not by pressing return at the prompt): 

(Se) Tiden > 

Joacim Melin skriker: Vansinnigt tråkigt väder ute!

(Se) Tiden >

**Comment by punktniklas** [comment]
How often are you actually logged in at the same time as at least two other persons these days..? :-)

How about a flag to "säg" instead? Like "säg -a" to send to all users. I don't like the name "skrik", it's almost the same as "skriv".

Whether it should be shown without pressing return or not is really a separate issue. It should be the same as for regular messages.  So if that's changed it should be changed for all types of messages.

**Comment by joacimmelin** [comment]
Actually, several NiKom BBs:s has more than one or two users logged in at the same time many times a day. 

Also, a command like this can increase the time spent logged in and enable more communication between users.

Maybe "skrik" isn't a good choice. How about "ropa" ?

**Comment by jayminer81** [comment]
I'm pretty sure 'ropa' is used for 'ropa på sysop', can't check to be absolutely sure right now. 

This is something that could be added very easily with a simple arexx-script and something I have been thinking of adding - but an official command for this would be nice and I like the idea of a flag to toggle if you want to see such messages.

**Comment by punktniklas** [comment]
Yes, there already is a command called "Ropa". I really think that this should be an addition to the existing "Säg" command.

There are really three separate issues here:
- Being able to send something to all logged in users
- More fine grained control over what kind of messages you want to receive
- Whether messages should be displayed at other times than when a prompt is displayed.

**Comment by joacimmelin** [comment]
If nothing else: if you just write "skri hej hopp" and it's not clear if you mean skrik or skriv, you can always display a message about that. NiKom does that now for other commands, such as: 

"li ny" which shows me: 
FLERTYDLIGT KOMMANDO

LISTA NYHETER
LISTA NYCKLAR

So "skrik" being to similiar to "skriv" isn't a problem I think.
